setup-env/action.yml

```diff
@@ -26,8 +26,6 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - uses: actions/checkout@v2
-
```

Comment:
What exactly was the issue here? Was this action clashing with something?

Reply:
Mainly it is to give the user more control. Perhaps they want to do something post-checkout but pre-deps. The point is the two shouldn't be coupled

Reply:
You'll see similar commits across lots of github actions after people use them for a bit and figure this out.

Reply:
https://github.com/yetanalytics/actions/pull/1/commits/d64f58204305b3b1aab18deb2a3a8a3fd9f80ceb